vipps-order-management-api-quick-start.md

```diff
@@ -65,7 +65,7 @@ Save the following files to your computer:
 
 ## Make API calls
 
-For all of the following, you will start by sending request `Get Access Token` which uses
+For all the following, you will start by sending request `Get Access Token` which uses
```

Comment:
isn't "For all of the following" correct here?

Reply:
I think this was suggested by LTeX to remove wordiness. I agree with you though. I like it better with 'of' as well.

Reply:
See here: https://community.languagetool.org/rule/show/ALL_OF_THE?lang=en-US

Reply:
But, it stops me up as well, so it will stop others. I will add 'of' back in.